terraform/layer1-aws/templates/userdata-amazonlinux2.tpl

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
```

Comment:
Why do we need this file? Where is it used?

Reply:
Agree

Reply:
fixed